qemu/tests/cfg/in_place_upgrade.cfg

```
@@ -0,0 +1,83 @@
+- in_place_upgrade:
+    only RHEL
+    pre_release = "release 8"
+    post_release = "release 9"
+    type = in_place_upgrade
+    login_timeout = 300
+    release_check = "cat /etc/redhat-release"
+    new_kernel_ver = "5.14"
+    fix_firewalld = "sed -i 's/AllowZoneDrifting=yes/AllowZoneDrifting=no/g' "
+    fix_firewalld += "/etc/firewalld/firewalld.conf"
+    fix_permit = "sed -i 's/PermitRootLogin yes/PermitRootLogin no/g' "
+    fix_permit += "/etc/ssh/sshd_config"
+    restore_permit = "sudo sed -i 's/PermitRootLogin no/PermitRootLogin yes/g'"
+    restore_permit += " /etc/ssh/sshd_config"
+    check_file = "sudo ls /var/log/leapp/answerfile"
+    restart_sshd = "sudo systemctl restart sshd.service"
+    add_ipuser = "useradd ipuser"
+    add_wheel_ipuser = "usermod -aG wheel ipuser"
+    add_passwd_ipuser = "echo ipuser |passwd --stdin ipuser"
+    no_passwd_for_sudo = "echo -e 'ipuser\tALL=(ALL)\tNOPASSWD: ALL' > "
+    no_passwd_for_sudo += "/etc/sudoers.d/leapp_ipuser_sudo"
+    check_kernel = "uname -r"
+    ipu_after_timeout = 6000
+    user_assistant = "ipuser"
+    user_assistant_pw = "ipuser"
+    reboot_cmd = "reboot"
+    yum_update = "yum -y update"
+    clean_up_old_kernel = "dnf -y remove --oldinstallonly"
+    depends_pkgs = "yum-utils"
+    error_info = "Error: No old installonly packages found for removal"
+    stop_yum_update = "rm -rf /var/run/yum.pid"
+    check_repo_list = "yum repolist enabled"
+    # <repo_leapp> it's leapp tool's repo
+    # <ins_leapp_cmd> install leapp tool command
+    # <prepare_env>, <get_answer_files_source> and <get_answer_files>
+    # private script to prepare in place upgrade env
+    # <old_custom_internal_repo>
+    # this parameter should contain the repo files, by which you can upgrade
+    # old system to the newer version before you really do in place upgade
+    # <new_rhel_content>
+    # this parameter should contain your upgraded system's repo files
+    # <pre_upgrade_no_rhsm>, <process_upgrade_no_rhsm>
+    # <pre_upgrade_rhsm>, <process_upgrade_rhsm>
+    # these paramters are leapp preupgrade and upgrade command
+    variants:
+        - without_rhsm_rhel8_to_rhel9:
+            rhsm_type = "no_rhsm"
+            pre_upgrade_no_rhsm = "leapp preupgrade --debug --no-rhsm"
+            process_upgrade_no_rhsm = "leapp upgrade --debug --no-rhsm"
+            disable_content = "yum-config-manager --disable "*""
```

I think you should use `'` to wrap `"`
```suggestion
            disable_content = 'yum-config-manager --disable "*"'
```

---

It's ok for me to change it, the origin also is acceptable in my opinions.

---

If the line is used for shell, this will definitely fail. It should be a bad habit and avoid using it in cartesian.

---

Updated, thanks a lot. :)